src/crimson/common/operation.h

```diff
@@ -20,6 +20,7 @@
 #include "include/utime.h"
 #include "common/Clock.h"
 #include "crimson/common/interruptible_future.h"
+#include "crimson/common/smp_helpers.h"
```

Comment:
Do we need the include in this particular commit (d8f1d193f0e493d6e6fc0b251f991eede9336f81)?

Reply:
Moved up.